spec/models/comment_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject{ described_class.new }
+  it "is valid with valid attributes" do
+    subject.body = "anything_body"
+    user = User.create!(email: "anything@io", password: "not_empty", password_confirmation: "not_empty")
```

Comment:
Instead of using `User.create!`, you can add user.rb to factories, and do something like:

```suggestion
    user = create(:user)
```